src/frontend/csrs-portal/src/app/components/landing/landing.component.scss

```diff
@@ -0,0 +1,589 @@
+@import "palette";
+@import "bootstrap/bootstrap-imports";
+
+h1 {
+  font-size: 3em;
+  line-height: 1.1em;
+  margin-bottom: 50px;
+  font-weight: bold;
+  text-shadow: 2px 2px 4px #000;
+}
+
+h3 {
+  font-size: 1.875em;
+  line-height: 1.4em;
+  font-weight: bold;
+  margin-bottom: 10px;
+  color: theme-palette(blue);
+}
+
+h4 {
+  font-size: 1.5em;
+  font-weight: bold;
+  margin-bottom: 20px;
+  color: theme-palette(blue);
+}
+
+.lead {
+  font-size: 1.5em;
+  margin-bottom: 10px;
+  line-height: 1.5em;
+  text-shadow: 2px 2px 4px #000;
+}
+
+@media (max-width: 320px) {
+  div.landing-summary {
+    background: url("/assets/landing-page-background-320x500.jpg") no-repeat !important;
```

Comment:
can you use image tag in html and put this image url so that we can change the size of image according to our need ?

Reply:
landing.scss file from landing component had several media tags, for example:
@media (min-width: 993px) {
div.landing-summary {
background: url("/assets/landing-page-background.png") no-repeat;
-webkit-background-size: cover;
-moz-background-size: cover;
-o-background-size: cover;
background-size: cover;
}
}
I used them to display background image on landing page. Kartheek had asked me why I did not use image tag directly in html file of landing component. It was good idea. I **removed media tags with background images from scss** file and developed

<div style="background-image: url('/assets/landing-page-background.png');
background-repeat: no-repeat;
background-position: top;
background-size: 100%;">
<div class="container"> in landing html file.

Reply:
<div style="background-image: url('/assets/landing-page-background.png');
background-repeat: no-repeat;
background-position: top;
background-size: 100%;">
<div class="container"> in landing html file.